sdk/search/azure-search-documents/pom.xml

```diff
@@ -22,6 +22,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <jacoco.min.linecoverage>0.70</jacoco.min.linecoverage>
+    <jacoco.min.branchcoverage>0.60</jacoco.min.branchcoverage>
```

Comment:
Is theere a follow up issue to track this?

Reply:
Nope as this is an increase over the normal coverage requirements of 0.40 and 0.30 respectively.